documentation/src/main/asciidoc/topics/search.adoc

```diff
@@ -0,0 +1,1087 @@
+[id='search_api']
+=  Indexing and Searching
+
```

Comment:
@gustavonalle Perhaps consider dropping the "Overview" heading or move the first sentence ("
{brandname} provides a search API...") under the top-level heading. Headers should always have some text underneath and not just float.